pkgs/applications/networking/browsers/mullvad-browser/default.nix

```
@@ -259,7 +297,7 @@ stdenv.mkDerivation rec {
     mainProgram = "mullvad-browser";
     homepage = "https://mullvad.net/en/browser";
     platforms = attrNames sources;
-    maintainers = with maintainers; [ felschr panicgh ];
+    maintainers = with maintainers; [ felschr panicgh heywoodlh ];
```

```suggestion
    maintainers = with maintainers; [ felschr heywoodlh panicgh ];
```
Nit: alphabetical order.